feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -122,10 +122,22 @@ private fun BookmarksGrid(
             .fillMaxSize()
             .testTag("bookmarks:feed")
     ) {
-        newsFeed(
-            feedState = feedState,
-            onNewsResourcesCheckedChanged = { id, _ -> removeFromBookmarks(id) },
+        items(
+            items = bookmarkItems,
+            key = BookmarkItem::key,
+            contentType = BookmarkItem::contentType,
```

Comment:
This change also akes it possible for use to use the `contentType` optimizations.